docs/dev/test-configuration-options.md

```diff
@@ -279,121 +306,249 @@ Specifies the screen resolution to be used during your test session. Default scr
+
+- For Android the value needs to be `"Chrome"`.
+- For iOS, the value needs to be `"Safari"`.
+
+:::note
+- If this capability is not provided for a virtual device, the ['app'](#app) capability needs to be set. If none is set the test will throw an error.
+- This capability can be omitted for virtual devices if the ['app'](#app) capability is set.
+- If this capability is not provided for a real device session and also the ['app'](#app) capability is not provided then a real device session will automatically default back to the default browser. This will be Chrome for Android and Safari for iOS
+:::
 
 ```java
-"appium:automationName": "UiAutomator2"
+MutableCapabilities capabilities = new MutableCapabilities();
+capabilities.setCapability("browserName", "chrome");
 ```
 
 ---
-### `appPackage`
-<p><small>| STRING | <span className="sauceDBlue">Android Only</span> |</small></p>
 
-Allows you to specify the Java package of the Android app you want to run.
+### `appium:app`
 
-:::tip Automatic Package Detection
+<p><small>| OPTIONAL | STRING | <span className="sauceDBlue">Virtual and Real Devices</span> |</small></p>
 
-Appium automatically determines the package to launch; you'll only need to use this capability if you want to specify a package different from the default one.
-:::
+Allows you to set a path to an `.ipa`, `.apk`, `.aab` or `.zip` file containing the mobile app you want to test. This could be the location of your app in [App Storage](/mobile-apps/app-storage) (e.g., `storage:filename=myapp.zip`) or the URL to a remote location where your app is located (e.g., `http://myappurl.zip`). The remote location needs to be accessible from the web, Sauce Connect can not access your internal file system where apps are hosted.
 
+:::note
+* If this capability is not provided for a virtual device, the ['browserName'](#browserName) capability needs to be set. If none is set the test will throw an error.
+* This capability can be omitted for virtual devices if the ['browserName'](#browserName) capability is set.
+* If this capability is not provided for a real device session and also the ['browserName'](#browserName) capability is not provided then a real device session will automatically default back to the default browser. This will be Chrome for Android and Safari for iOS.
+:::
 
 ```java
-"appium:appPackage": "com.example.android.myApp, com.android.settings"
+MutableCapabilities capabilities = new MutableCapabilities();
+capabilities.setCapability("appium:app", "storage:filename=my_app.zip");
 ```
 
+:::tip Using Storage Id
+If you have an app you have uploaded to [Sauce storage](https://app.saucelabs.com/live/app-testing), you can also set the `app` capability to `"storage:xxxxxxxxx-xxxxxxx-xxx"` and enter the **FILE ID** for your app. This allows you to set which specific version you uploaded, otherwise if you use the file name, it will use the latest one uploaded with the exact same name.
+:::
+
```

Comment:
```suggestion
:::tip Using Storage Id
If your app has been uploaded to [Sauce storage](https://app.saucelabs.com/live/app-testing), you can set the `app` capability to `"storage:xxxxxxxxx-xxxxxxx-xxx"` and enter the **FILE ID** for your app. This allows you to set which specific version you uploaded. Otherwise, if you use the file name it will select the latest version uploaded with the exact same name.
:::

```